julia/src/model.jl

```diff
@@ -57,7 +59,7 @@ mutable struct StanModel
 
         function f(sm)
             ccall(
-                Libc.Libdl.dlsym(sm.lib, "destruct"),
+                Libc.Libdl.dlsym(sm.lib, "bs_destruct"),
```

Comment:
`construct` -> `bs_construct` (line 47).

Reply:
Good catch, I did a search for the string `sm.lib` but in the constructor itself the library is called only `lib`. I assume the tests would have caught it rather quickly, but I only ran the Python ones locally before pushing